File: Client/src/models/Convalidation.ts
export interface Convalidation {
  id:                    number;
  rol:                   string;
  id_origin_course:      number;
  id_destination_course: number;
  state:                 string;
  comments:              string | null;
  creation_date:         string;
  approval_date:         string | null;
  user_approves:         number;
}

File: Server/models/Convalidation.py
from typing import Optional, Text
from datetime import datetime
from pydantic import BaseModel

class Convalidation(BaseModel):
    id: int
    rol: str
    id_origin_course: int
    id_destination_course: int
    state: str
    comments: Optional[Text]
    creation_date: datetime
    approval_date: Optional[datetime]
    user_approves: int
